Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -173,18 +173,9 @@ void UpdateAll()
 			UpdateTextColor();
 			UpdateEnabled();
 			UpdateBackgroundColor();
-			UpdateDrawable();
 			UpdatePadding();
 		}
 
-		void UpdateDrawable()
-		{
-			if (Element == null || Control == null)
-				return;
-
-			_backgroundTracker?.UpdateDrawable();
```

Comment:
You can also get rid of the method on `ButtonBackgroundTracker`, since nothing calls it now. I wonder if we had a reason for adding this, like we wanted to be sure the Drawable gets initialized, even if a custom renderer overrides `UpdateBackgroundColor` and removes the call to the tracker? @rmarinho, do you happen to remember? #941

Reply:
Good point. I applied this same fix to all the renderers and just removed the redundant update call inside BackgroundTracker. I am curious about your question. It almost feels like ensuring it gets initialized despite the override would be more of a hostile experience though then a positive one. Seems like you'd want to be able to let them override and not initialize the drawable